app/src/pages/admin/projectsPage/projectsPage.jsx

```diff
@@ -60,7 +61,8 @@ const HEADER_BUTTONS = [
 
 @connect(
   (state) => ({
-    projectId: projectIdSelector(state),
+    projectKey: urlProjectKeySelector(state),
+    organizationSlug: projectOrganizationSlugSelector(state),
```

Comment:
Should be selected also from the URL to be consistent.

Reply:
Done.